src/conn/DBConnection.h

```diff
@@ -30,10 +30,10 @@ class DBConnection
     
     private:
         //Connection Variables
-        sql::Connection *conn;
-        sql::PreparedStatement *prep_stmt;
+        sql::Connection *_conn;
+        sql::PreparedStatement *_prep_stmt;
```

Comment:
They are protected by scope. Even if they were both public, they wouldn't clash as they are class attributes, not global variables i.e. DBConnection._connection cannot clash with Model._connection